JupyterDocs/ariaMisclosure/ariaMisclosure_tutorial.ipynb

```diff
@@ -1,10 +1,5 @@
 {
```

Comment:
<p>Do you need to keep the " " and put key on between or not? clarify</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aria-tools/ARIA-tools-docs/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Hmm no you can just do <code>echo KEY &gt; ~/.topoapi</code></p><p>Is this what you meant? </p>